source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -728,6 +730,14 @@ eagerly so that any operations waiting on `Connections <#connection>`_ can retry
 as soon as possible. The pool MUST NOT rely on WaitQueueTimeoutMS to clear
 requests from the WaitQueue.
 
+The clearing method(s) MUST provide a way to close perished connections as part 
```

Comment:
I think the wording of this paragraph was a bit clearer before, can we revert the most recent changes to it?